nemo_skills/dataset/gpqa/__init__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Please add gpqa in here https://github.com/NVIDIA/NeMo-Skills/blob/main/tests/test_datasets.py#L20

Reply:
done